src/connectors/siriusxm-player.ts

```diff
@@ -10,34 +10,47 @@ export {};
 const filter = MetadataFilter.createFilter({ track: removeYear });
 const removeYearRe = /\s\(\d{2}\)\s?$/g;
 
-Connector.playerSelector = '.sxm-player-controls';
-
-Connector.artistSelector = '.sxm-player-controls .artist-name';
-
-Connector.trackSelector = '.sxm-player-controls .track-name';
+Connector.playerSelector = ".sxm-player-controls";
+Connector.artistSelector = ".sxm-player-controls .artist-name";
+Connector.trackSelector = ".sxm-player-controls .track-name";
 
 Connector.isPlaying = () => {
 	return (
 		Util.getAttrFromSelectors(
-			'.sxm-player-controls .play-pause-btn',
-			'title'
-		) === 'Pause'
+			".sxm-player-controls .play-pause-btn",
+			"title"
+		) === "Pause"
 	);
 };
 
-Connector.trackArtSelector = '.album-image-cell img';
+Connector.trackArtSelector = ".album-image-cell img.album-image";
 
 Connector.isScrobblingAllowed = () => {
-	const artist = Connector.getArtist();
-	if (artist) {
-		return !artist.toLowerCase().includes('siriusxmu');
-	}
-
-	return false;
+	const artist = Connector.getArtist()?.toLowerCase();
+	const track = Connector.getTrack()?.toLowerCase();
+	const filteredTerms = [
+		"@siriusxm",
+		"@jennylsq",
+		"@radiomadison",
+		"@morningmashup",
+		"josiah",
+		"1-877-33-sirius",
+		"@sxm", //will broadly catch a bunch of sxm Twitter handles
+		"altnation",
+		".c", // will catch .com and .ca URLs
```

Comment:
Would it be better to be more explicit and have two entries for `.ca` and `.com`?

Reply:
@inverse I can do that. Probably getting too cute doing that anyways 😂

Reply:
@inverse I pushed that change over